plottr/apps/inspectr.py

```diff
@@ -142,6 +145,21 @@ def __init__(self, parent: Optional[QtWidgets.QWidget] = None):
         self.itemSelectionChanged.connect(self.selectRun)
         self.itemActivated.connect(self.activateRun)
 
+        self.setContextMenuPolicy(QtCore.Qt.CustomContextMenu)
```

Comment:
I cannot see that QtCore is ever imported into this module

Reply:
it is already imported at the top :)